src/components/image/image.tsx

```diff
@@ -112,40 +112,59 @@ export class EuiImage extends Component<EuiImageProps, State> {
     const fullScreenDisplay = (
       <EuiOverlayMask onClick={this.closeFullScreen}>
         <EuiFocusTrap clickOutsideDisables={true}>
-          <button
-            type="button"
-            className="euiImage-isFullScreen"
-            onClick={this.closeFullScreen}
-            onKeyDown={this.onKeyDown}>
-            <figure>
-              <img src={url} className="euiImage-isFullScreen__img" alt={alt} />
-              {optionalCaption}
-
+          <figure
+            className="euiImage euiImage--isFullScreen"
```

Comment:
This is still technically a state modifier so you'd still use the single dash like before
```suggestion
            className="euiImage euiImage-isFullScreen"
```